packages/grafana-ui/src/components/index.ts

```diff
@@ -145,6 +145,8 @@ export { Field } from './Forms/Field';
 export { Legend } from './Forms/Legend';
 export { FieldSet } from './Forms/FieldSet';
 export { FieldValidationMessage } from './Forms/FieldValidationMessage';
+export { InlineField } from './Forms/InlineField';
```

Comment:
Shouldn't `InlineFieldRow` also be exported here?

```suggestion
export { InlineField } from './Forms/InlineField';
export { InlineFieldRow } from './Forms/InlineFieldRow';
```

Reply:
Good catch! Missed that one.